silkrpc/consensus/ethash.cpp

```diff
@@ -27,7 +27,7 @@ BlockReward compute_reward(const ChainConfig& config, const silkworm::Block& blo
         throw std::runtime_error("Invalid chain config");
     }
     const auto chain_config = cc_optional.value();
-    const auto revision = chain_config.revision(block.header.number);
+    const auto revision = chain_config.revision(block.header.number, 0);
```

Comment:
Please add a comment to make the second actual parameter more readable:
```
const auto revision = chain_config.revision(block.header.number, /*block_time=*/0);
```

Reply:
ok